xwiki-platform-core/xwiki-platform-edit/xwiki-platform-edit-ui/src/main/resources/XWiki/InplaceEditing.xml

```diff
@@ -817,8 +817,7 @@ define('editInPlace', [
       // Check for merge conflicts only if the document is not new and we know the current version.
       if (!xwikiDocument.isNew &amp;&amp; xwikiDocument.version) {
         formData.previousVersion = xwikiDocument.version;
-        // It would have been easier to send the timestamp but that's what the Save action expects.
-        formData.editingVersionDate = new Date(xwikiDocument.modified).toISOString();
+        formData.editingVersionDate = new Date(xwikiDocument.modified).getTime();
```

Comment:
``xwikiDocument.modified`` is already a timestamp AFAICS, but that's really minor.

Reply:
I didn't check it back tbh, at least it's obvious by reading the code that we get a timestamp there.